src/core/bpf/meson.build

```diff
@@ -14,11 +14,13 @@ bpf_clang_flags = [
 ]
 
 bpf_gcc_flags = [
+        '-std=gnu11',
```

Comment:
How about to also set the same option for clang in the above??

Reply:
AFAIU clang doesn't need this, gcc sometimes needs it for `typeof` support to work, there's an alternate `__typeof__` style which works with regular c11 in gcc as well but bpf headers uses mostly the `typeof` style.

Reply:
hmm, actually looks like might be helpful in clang(I guess defaults work however)

Reply:
added to clang flags as well

Reply:
Even so, I think it is better to add that for consistency, if adding the option for clang does not introduce any issue. Then, all codes are compiled with `gnu11`.

Reply:
> added to clang flags as well

Thanks.